src/app/static/src/tests/components/runReport/runReport.test.ts

```diff
@@ -228,7 +318,7 @@ describe("runReport", () => {
                 metadata: {
                     git_supported: true
                 },
-                gitBranches
+                initialGitBranches
             },
             data() {
                 return {
```

Comment:
@carbowizzy any idea why the test at line 344 might be failing? It's just receiving an empty array rather than the `emittedParams` object
`expect(wrapper.vm.$data.parameterValues).toMatchObject(emittedParams)`